tests/vpp.cpp

```diff
@@ -21,18 +21,22 @@
 #include "vppoutputencode.h"
 #include "encodeinput.h"
 #include "common/log.h"
+#include "tests/vppinputasync.h"
 #include <Yami.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <getopt.h>
 #include <string.h>
+#include <deque>
 
 using namespace YamiMediaCodec;
 
 void usage();
 
-SharedPtr<VppInput> createInput(const char* filename, const SharedPtr<VADisplay>& display)
+SharedPtr<VppInput> createInput(const char* filename, const SharedPtr<VADisplay>& display, uint32_t startupSize)
```

Comment:
It should be used to specific the frame number:

> bool VppInputAsync::preloadFrames(uint32_t startupSize)
{
    uint32_t size;
    for (size = 0; size < startupSize; size++) {
        SharedPtr<VideoFrame> frame;
        if (!m_input->read(frame))
            break;
        m_queue.push_back(frame);
    }

So uint32_t may be enough.